pkg/oci/oci.go

```diff
@@ -58,9 +58,14 @@ type VerifyOptions struct {
 	PublicKey       string
 }
 
+type AuthorizedDigestsOptions struct {
+	AuthorizedDigests []string
```

Comment:
Authorized -> Allowed: because in this context, we are not authorized by an external authority. It is just the client deciding what is allowed or not.

```suggestion
	AllowedDigests []string
```